src/UserDetailsPopover/UserDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import { joinNonEmpty } from '@mll-lab/js-utils';
+import React, { useMemo } from 'react';
+
+import { UserAvatar } from '../Avatar';
+import { Divider } from '../Divider';
+import { Col, Row } from '../Grid';
+import { Space } from '../Space';
+import { Typography } from '../Typography';
+
+import { EmailRow } from './EmailRow';
+import { GUTTER, SPACE } from './constants';
+
+type User = {
+  acronym?: string | null;
+  email?: string | null;
+  firstName?: string | null;
+  inactive?: boolean;
+  lastName?: string | null;
+  userName: string;
```

Comment:
At least `username` is commonly spelled as one word, and we do the same with `firstname` and `lastname`.